### DISCUSSION TITLE
How do I get my bash PS1?

### DISCUSSION THREAD

**Discussion by kaihendry** [starship/starship#4421] [Q&A] [answered]
My .bashrc looks like:

```
PS1='[\u@\h \W]\$ '
eval "$(starship init bash)"
```

Which ends up looking like:

![image](https://user-images.githubusercontent.com/765871/193195957-f1db2968-3cdb-4e99-8aca-a85f9eb09a25.png)

PS1 has no affect? How do I fix this?

**Comment by matchai** [comment, marked as answer]
Starship will override your PS1 as long as it is initialized.
If you'd like your PS1 to take effect, remove the line initializing Starship.

**Comment by kaihendry** [comment]
Don't quite understand the @ symbol on the default with _no configuration_ on Archlinux

![image](https://user-images.githubusercontent.com/765871/193437175-afc86b95-4643-40ae-9d83-0f2c6ed9edd0.png)

**Reply by matchai** [reply]
Running `starship explain` should break down which module is showing the "@" symbol.

**Reply by kaihendry** [reply]
```
@❯ starship explain

 Here's a breakdown of your prompt:
 "~ " (<1ms)  -  The current working directory
 "❯ " (<1ms)  -  A character (usually an arrow) beside where the text is entered in your terminal

~
@❯
```

It doesn't explain the at symbol 😭

**Reply by DrHyde** [reply]
The whole "on master" thing is a bit odd if that's what `starship explain` says too!

**Reply by kaihendry** [reply]
The @ symbol was caused by `set show-mode-in-prompt on` in my ~/.inputrc 🤦
